src/test/java/com/company/LocationsTest/LocationsTest.java

```diff
@@ -0,0 +1,74 @@
+package com.company.LocationsTest;
+
+import com.company.controller.LocationController;
+import com.company.model.dto.SaveLocationDTO;
+import com.company.model.entity.Location;
+import org.junit.Assert;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import java.util.List;
+import static org.junit.Assert.assertNotNull;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@SpringBootTest
+public class LocationsTest {
+
+    @Autowired
+    private LocationController locationController;
+
+
+    @Test
+    void testFindAll(){
+        ResponseEntity<List<Location>> response = locationController.findAll();
+        List<Location> locations = response.getBody();
+
+        assertNotNull(locations);
+        Assert.assertEquals(HttpStatus.OK, response.getStatusCode());
+    }
+
+    @Test
+    void testSave() {
+        SaveLocationDTO location = new SaveLocationDTO("CountryName", "CityName", "StateName");
+        ResponseEntity<Location> saveLocation = locationController.save(location);
+        Assert.assertEquals("CountryName", saveLocation.getBody().getCountry());
+        Assert.assertEquals("CityName", saveLocation.getBody().getCity());
+        Assert.assertEquals("StateName", saveLocation.getBody().getState());
```

Comment:
Lets change the Assert methods for the static methods in Assertions. Check how the other tests are done